examples/Bickley_jet_shallow_water.jl

```diff
@@ -102,9 +89,9 @@ model = ShallowWaterModel(
 # decays away from the center of the jet.
```

Comment:
Perhaps rephrase to

"The initial conditions have a small-amplitude random perturbation that decays away from the center of the jet."